content/articles/2018-08-05-junit-5-extension-model.md

```diff
@@ -296,12 +296,12 @@ If you have an extension that you think needs to be registered with all tests in
 Simply let your extension JAR proclaim that it provides implementations of `org.junit.jupiter.api.extension.Extension` and Jupiter picks it up.
 
 Almost... Automatic registration is turned off by default, so you first need to [configure Jupiter](https://junit.org/junit5/docs/current/user-guide/#running-tests-config-params) to auto-detect extensions by setting `junit.jupiter.extensions.autodetection.enabled` to `true`.
-While your at it, consider requiring explicit activation for your extension with your own parameter (you can query it with the store's `getConfigurationParameter` method).
+While you're at it, consider requiring explicit activation for your extension with your own parameter (you can query it with the store's `getConfigurationParameter` method).
 This way you can use your extension JAR without all global extensions being registered all the time.
 
 ### Custom Annotations
 
-The JUnit Jupiter API is driven by annotations and the engine does a little extra work when it checks for their presence: It not only looks for annotations on classes, methods and parameters but also *on other annotations*.
+The JUnit Jupiter API is driven by annotations, and the engine does a little extra work when it checks for their presence: it looks for annotations not only on classes, methods and parameters but also *on other annotations*.
```

Comment:
Oh, I have to remember this one. I think I make this mistake quite frequently (when I use that phrase).